utils/cosmos/iscn/sign.ts

```diff
@@ -25,31 +26,63 @@ export function formatISCNTxPayload(payload: ISCNRegisterPayload): ISCNSignPaylo
     authorNames,
     authorUrls,
     authorWallets,
+    likerIds,
+    descriptions,
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     cosmosWallet,
     ...data
   } = payload;
+
   const contentFingerprints = []
   if (fileSHA256) contentFingerprints.push(`hash://sha256/${fileSHA256}`)
   if (ipfsHash) contentFingerprints.push(`ipfs://${ipfsHash}`)
-  const stakeholders = []
+  const stakeholders: any = []
   if (authorNames.length) {
     for (let i = 0; i < authorNames.length; i += 1) {
-      const authorName = authorNames[i]
-      const authorUrl = authorUrls[i]
-      const authorId = authorWallets[i]
-      const isNonEmpty = authorUrl || authorName || authorId
+      const authorName: string = authorNames[i]
+      const sameAs: any = []
+      const identifier: any = []
```

Comment:
Prefer naming as `sameAsArray` `identifiers`, then when putting into object use the original name as key e.g. `identifier: identifiers`